hls4ml/backends/vivado_accelerator/vivado_accelerator_backend.py

```diff
@@ -52,6 +88,7 @@ def create_initial_config(self, board='pynq-z2', part=None, clock_period=5, io_t
         config['AcceleratorConfig']['Board'] = board
         config['AcceleratorConfig']['Interface'] = interface  # axi_stream, axi_master, axi_lite
         config['AcceleratorConfig']['Driver'] = driver
+        config['AcceleratorConfig']['Platform'] = platform
```

Comment:
Could we only set this if the `board` requires it (i.e. is Alveo)?
Otherwise it could return something confusing like:
`{'Board' : 'pynq-z2',
   'Platform' : 'xilinx_u250_xdma_201830_2'}